routers/routes/routes.go

```diff
@@ -622,6 +624,19 @@ func RegisterRoutes(m *macaron.Macaron) {
 		}, context.RepoIDAssignment(), context.UnitTypes(), reqRepoCodeReader)
 	}, reqSignIn)
 
+	m.Group("/projects", func() {
+		m.Get("/create", repo.CreateProject)
+		m.Post("/create", bindIgnErr(auth.UserCreateProjectForm{}), repo.CreateProjectPost)
+	}, repo.MustEnableProjects, func(ctx *context.Context) {
+
+		if err := ctx.User.GetOrganizations(&models.SearchOrganizationsOptions{All: true}); err != nil {
```

Comment:
What is this about?

The first line of CreateProjectPost uses checkContextUser which sets ctx.Data["Orgs"] to a different value.

Reply:
since Projects for Org are not implemented jet we can just remove this - will test if it afects something

Reply:
@zeripath removed